modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -317,10 +325,124 @@ Status description:: error: transaction rollback - constituent rollback failed.
 
 Syntax error or access rule violation errors occur when a Cypher query contains invalid syntax or when a client request violates the access rules, such as when a query tries to access a database without enough privileges, etc.
 
+=== 42000
+
+Status description:: error: syntax error or access rule violation
+
+=== 42001
+
+Status description:: error: syntax error or access rule violation - invalid syntax
+
+=== 42002
+
+Status description:: error: syntax error or access rule violation - invalid reference
+
+=== 42004
+
+Status description:: error: syntax error or access rule violation - use of visually confusable identifiers
+
+=== 42006
+
+Status description:: error: syntax error or access rule violation - number of edge labels below supported minimum
+
+=== 42007
+
+Status description:: error: syntax error or access rule violation - number of edge labels exceeds supported maximum
+
+=== 42008
+
+Status description:: error: syntax error or access rule violation - number of edge properties exceeds supported maximum
+
+=== 42009
+
+Status description:: error: syntax error or access rule violation - number of node labels below supported minimum
+
+=== 42010
+
+Status description:: error: syntax error or access rule violation - number of node labels exceeds supported maximum
+
+=== 42011
+
+Status description:: error: syntax error or access rule violation - number of node properties exceeds supported maximum
+
+=== 42012
+
+Status description:: error: syntax error or access rule violation - number of node type key labels below supported minimum
+
+=== 42013
+
+Status description:: error: syntax error or access rule violation - number of node type key labels exceeds supported maximum
+
+=== 42014
+
+Status description:: error: syntax error or access rule violation - number of edge type key labels below supported minimum
+
+=== 42015
+
+Status description:: error: syntax error or access rule violation - number of edge type key labels exceeds supported maximum
+
+=== 42I00
+
+Status description:: error: syntax error or access rule violation - invalid case expression. 'CASE' expressions must have the same number of 'WHEN' and 'THEN' operands.
+
+=== 42I01
+
+Status description:: error: syntax error or access rule violation - invalid FOREACH. Invalid use of `{ $clause }` inside 'FOREACH'.
+
+=== 42I02
+
+Status description:: error: syntax error or access rule violation - invalid comment. Failed to parse comment. A comment starting with `'/*'` must also have a closing `'*/'`.
+
+=== 42I06
+
+Status description:: error: syntax error or access rule violation - invalid input. Invalid input `{ $input1 }`, expected: `{ $input2 }`.
+
+=== 42I18
+
+Status description:: error: syntax error or access rule violation - invalid reference to implicitly grouped expressions. The aggregation column contains implicit grouping expressions referenced by the variables `{ $varList }`. Implicit grouping expressions are variables not explicitly declared as grouping keys.
```

Comment:
```suggestion
Status description:: error: syntax error or access rule violation - invalid reference to implicitly grouped expressions. The aggregation column contains implicit grouping expressions referenced by the variables `{ $variableList }`. Implicit grouping expressions are variables not explicitly declared as grouping keys.
```

Same thing with var vs. variable as I mentioned on another PR. I have updated this one and the other cases of var/varList in this PR in the spreadsheet too.